build/ci/install_ci_dependencies.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+set +o pipefail
+
+# install gh command
+curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+sudo apt-get update
+sudo apt-get install gh
```

Comment:
this seems to be missing a newline at EOF

Reply:
done